plugins/maven/plugin/pom.xml

```diff
@@ -69,14 +69,26 @@
       <version>3.0</version>
       <scope>compile</scope>
     </dependency>
+
+    <dependency>
+        <groupId>com.mobidevelop.robovm</groupId>
+        <artifactId>robovm-debugger</artifactId>
+        <version>${project.version}</version>
```

Comment:
please remove version from here and add reference to debugger in  <dependencyManagement> of plugins/maven/pom.xml

Reply:
Done in 8494951.